website/content/docs/enterprise/license/autoloading.mdx

```diff
@@ -0,0 +1,29 @@
+---
+layout: docs
+page_title: License Autoloading
+description: An overview of license autoloading.
+---
+
+Prior to Vault 1.8, Vault Enterprise would be licensed using special binaries
+that contained embedded licenses, or via a license written into Vault storage
+using the `PUT sys/license` API.
```

Comment:
Is it worth linking to said API here?

Reply:
I was going to say no, but it was taking longer to justify than it would to just make the link.